pkgs/applications/window-managers/sway/wrapper.nix

```diff
@@ -27,7 +28,7 @@ let
        export DBUS_SESSION_BUS_ADDRESS
        exec ${sway}/bin/sway "$@"
      else
-       exec ${dbus}/bin/dbus-run-session ${sway}/bin/sway "$@"
+       exec ${if !dbusSupport then "" else "${dbus}/bin/dbus-run-session"} ${sway}/bin/sway "$@"
```

Comment:
optionalString would have been better https://github.com/NixOS/nixpkgs/blob/f8073ff44c8d06a354622cf5ced7d9d854c61f2a/lib/strings.nix#L198-L213

Reply:
Thank you; I will use that next time.